components/RegisterISCN.vue

```diff
@@ -78,21 +78,24 @@ const totalFee = computed(() => {
 
 const { payload } = useISCN(iscnData)
 
-const isFormValid = computed(() => {
+const formError = computed(() => {
   const requiredFields = {
     title: !!iscnData.value.title,
     description: !!iscnData.value.description,
     authorName: !!iscnData.value.author.name,
     contentUrl: !!iscnData.value.contentFingerprints.some(f => !!f.url)
   }
 
-  error.value = Object.entries(requiredFields)
-    .find(([_, isValid]) => !isValid)?.[0]
-    ?.toUpperCase() || ''
-
-  return Object.values(requiredFields).every(Boolean)
+  return Object.entries(requiredFields)
+    .find(([_, isValid]) => !isValid)?.[0]?.toUpperCase() || ''
 })
 
+const isFormValid = defineModel<boolean>('valid')
```

Comment:
Sorry for changing back-and-forth, but since `defineModel` allow parent to modify this variable, maybe we should just stick with `ref` + `watch` + `emit`